src/js/apps/forms/form/form_app.js

```diff
@@ -98,22 +93,30 @@ export default App.extend({
   onClickPrintButton() {
     Radio.request(`form${ this.form.id }`, 'print:form');
   },
-  onChangeSidebar(state, sidebar) {
-    this.stopChildApp('patient');
-    Radio.request('sidebar', 'close');
+  onChangeState(model) {
+    if (model.hasChanged('isExpanded') || model.hasChanged('isActionSidebar')) {
+      const isActionSidebar = model.get('isActionSidebar');
+      const isExpanded = model.get('isExpanded');
 
-    if (!sidebar) {
-      this.getRegion('sidebar').empty();
-      return;
-    }
+      this.startChildApp('patient', {
```

Comment:
I think this `startChildApp` should go after the `isExpanded`.  Really it can probably be the very last line, but the `if (isActionSidebar` wouldn't `return;`